src/Compilers/VisualBasic/Test/Emit/CodeGen/CodeGenAsyncTests.vb

```diff
@@ -8311,7 +8311,7 @@ Imports System
 Imports System.Collections
 Class C
     Shared Function F() As Object
-        Throw New ArgumentException()
+        Throw New ArgumentException("Value does not fall within the expected range.")
     End Function
```

Comment:
On Windows with Russian language pack, the default exception message is in Russian, so I provide it explicitly.